docs/source/en/model_doc/audioflamingo3.md

```diff
@@ -0,0 +1,165 @@
+*This model was released on 2025-07-10 and added to Hugging Face Transformers on 2025-10-15.*
+
+# Audio Flamingo 3
+
+<div class="flex flex-wrap space-x-1">
+<img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
+<img alt="FlashAttention" src="https://img.shields.io/badge/%E2%9A%A1%EF%B8%8E%20FlashAttention-eae0c8?style=flat">
+<img alt="SDPA" src="https://img.shields.io/badge/SDPA-DE3412?style=flat&logo=pytorch&logoColor=white">
+</div>
+
+## Overview
+
+Audio Flamingo 3 (AF3) is a fully open large audio–language model designed for robust understanding and reasoning over speech, environmental sounds, and music. AF3 pairs a Whisper-style audio encoder with a causal language model and performs replace-in-place audio–text fusion: the processor aligns post-pool audio frames to a dedicated placeholder token (written as `<sound>` in code) and the model replaces those token slots with projected audio embeddings during the forward pass.
+
+Highlights:
+
+- Unified audio encoder across speech, sound, and music.
+- Long-audio support via windowing and post-pool alignment (up to 10 minutes).
+- Deterministic fusion that preserves sequence length by replacing `<sound>` tokens with audio embeddings.
+
+### Paper
+
+[Audio Flamingo 3](https://huggingface.co/papers/2507.08128): Advancing Audio Intelligence with Fully Open Large Audio Language Models  
+A. Goel, S. Ghosh, J. Kim, S. Kumar, Z. Kong, S. Lee, C.-H. H. Yang, R. Duraiswami, D. Manocha, R. Valle, B. Catanzaro  
+NVIDIA and University of Maryland  
+Project: https://research.nvidia.com/labs/adlr/AF3/
+
+## Quickstart
+
+```python
```

Comment:
How about just a short text above explaining the example snippet? (Batch processing that does both transcription and music understanding)

Reply:
and another short example just above that shows just transcription with a single example. In general people like seeing multiple examples for different tasks so they can conveniently copy-paste